telluric/georaster.py

```diff
@@ -731,11 +731,38 @@ def width(self):
         """Raster width."""
         return int(self.shape[2])
 
+    @property
+    def source_file(self):
+        """ When using open, returns the filename used
+        """
+        return self._filename
+
     @property
     def height(self):
         """Raster height."""
         return int(self.shape[1])
 
+    def _add_overviews_and_tags(self, r, tags, kwargs):
+            # write tags:
+        tags_to_save = {'telluric_band_names': json.dumps(self.band_names)}
+        if tags:
+            tags_to_save.update(tags)
+
+        r.update_tags(**tags_to_save)  # in default namespace
+
+        # overviews:
+        overviews = kwargs.get('overviews', True)
+        resampling = kwargs.get('resampling', Resampling.cubic)
+        if overviews:
+            factors = kwargs.get('factors')
+            if factors is None:
+                factors = self._overviews_factors()
+            else:
+                factor_max = max(self._overviews_factors(blocksize=1), default=0)
```

Comment:
should we use `self.overviews_factors` here?

Reply:
no, the property that I wrote is not calculating, is reading the overview, this property is calculating the overviews